### ISSUE TITLE
Some issues with Vue Router's Named Views

### ISSUE THREAD

**Issue by davidangelesu** [Kitware/trame-router#2]
Hi there, I am having some issues with utilizing Named Vue's (maybe I might be trying to do something that is currently not supported)

I am currently defining some RouterViewLayout's:
 ```
 with RouterViewLayout(server, "/",name="Sidebar"):
    with vuetify.VCard():
        vuetify.VCardTitle("This is Sidebar")

with RouterViewLayout(server, "/", name="Content"):
    with vuetify.VCard():
        vuetify.VCardTitle("This is Content")
```

and I am trying to load a specific View in the drawer and another  one in the Sidebar
```
with SinglePageWithDrawerLayout(server) as layout:
    layout.title.set_text("Multi-View demo")

    with layout.content:
        with vuetify.VContainer():
            router.RouterView(name="Content")

    with layout.drawer:
        router.RouterView(name="Sidebar")

```
 
After some debugging it seems that the issue comes from the name parameter that i am passing to the RouterView is not being taken into consideration

**Comment by jourdain** [comment]
The name part for the router was never added, but it should be easy to enable.

**Comment by jourdain** [comment]
Are you using vue3 or vue2?

**Comment by davidangelesu** [comment]
Hi, I am currently using vue2

**Comment by jourdain** [comment]
Fixed in v2.2.0

**Comment by davidangelesu** [comment]
Amazing! thanks
